src/lib/db/access-store.ts

```diff
@@ -448,4 +449,17 @@ export class AccessStore implements IAccessStore {
             [destinationEnvironment, sourceEnvironment],
         );
     }
+
+    async isChangeRequestsEnabled(
+        project: string,
+        environment: string,
+    ): Promise<boolean> {
+        const result = await this.db.raw(
+            `SELECT EXISTS(SELECT 1 FROM ${T.CHANGE_REQUEST_SETTINGS}
```

Comment:
why do we need double select? can't we just do one and check it a result or null was returned and map it to boolean?